src/autopas/LogicHandler.h

```diff
@@ -648,10 +648,10 @@ class LogicHandler {
    * @return
    */
   template <class Functor>
-  void iterateRemainder(Functor &functor, bool newton3);
+  void computeRemainderInteractions(Functor &functor, const bool newton3);
```

Comment:
```suggestion
  void computeRemainderInteractions(Functor &functor, bool newton3);
```

`const` in function declarations for parameters passed by value has no effect. You can put them in function definitions but I wouldn't for consistency.